docs/how-to-guides/data-basics/1-data-basics-setup.md

```diff
@@ -8,33 +8,39 @@ Also, this tutorial will lean heavily on Redux concepts such as state, actions,
 
 We'll do all the development inside of a WordPress plugin. Let's start by creating a `wp-content/plugins/my-first-gutenberg-app` directory in your local WordPress environment. We will need to create four files inside that directory:
 
-* my-first-gutenberg-app.php – to create a new admin page
-* src/index.js – for our JavaScript application
-* style.css – for the minimal stylesheet
-* package.json – for the build process
+-   my-first-gutenberg-app.php – to create a new admin page
```

Comment:
I have Prettier integrated with my IDE and this is why these changes got included. It's also surprising because all Markdown files were formatted with Prettier a few months ago...

Reply:
Yeah, that's kinda weird 🤔  I hope that we don't end up accidentally using different configs for prettier/markdown in our scripts and in our editor configs.

Anyway, it's just whitespace 😬  Even if this isn't the format we want, it wouldn't be the end of the world.